pkgs/tools/graphics/map-machine/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, fetchFromGitHub, python3, pkgs }:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "map-machine";
+  version = "dcf3ffa00902d7812dd581c959e97d8d33f3a78d";
```

Comment:
Please try to ping the maintainer to get a tagged release. In the mean time, the usual convention is to name the version "unstable-YYYY-MM-DD" where you set the commit date as part of the version.